appservice/src/main/java/nu/yona/server/messaging/rest/GoalConflictMessageDecorator.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
+ * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.messaging.rest;
+
+import nu.yona.server.analysis.rest.BuddyActivityController;
+import nu.yona.server.analysis.rest.UserActivityController;
+import nu.yona.server.analysis.service.DayActivityDto;
+import nu.yona.server.analysis.service.GoalConflictMessageDto;
+import nu.yona.server.goals.rest.ActivityCategoryController;
+import nu.yona.server.messaging.rest.MessageController.MessageResourceAssembler;
+import nu.yona.server.messaging.service.MessageDto;
+
+@Decorates(GoalConflictMessageDto.class)
+public class GoalConflictMessageDecorator implements Decorator
+{
+	@Override
+	public void decorate(MessageResourceAssembler assembler, MessageDto message)
+	{
+		GoalConflictMessageDto goalConflictMessage = (GoalConflictMessageDto) message;
+		addActivityCategoryLink(assembler, goalConflictMessage);
+		addDayActivityDetailLink(assembler, goalConflictMessage);
+
+	}
+
+	private void addActivityCategoryLink(MessageResourceAssembler assembler, GoalConflictMessageDto message)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "assembler". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS1172)